addons/point_of_sale/static/src/js/models.js

```diff
@@ -2534,6 +2534,7 @@ export class Order extends PosModel {
         } else {
             this.sequence_number = this.pos.pos_session.sequence_number++;
             this.access_token = uuidv4(); // unique uuid used to identify the authenticity of the request from the QR code.
+            this.ticket_code = this._generate_ticket_code(); // 5-digits alphanum code shown on the receipt
```

Comment:
we're slowly trying to unify all those js convention styles into one. Could you change those into camelCase ?